html/changelog.html

```diff
@@ -1,57 +1,57 @@
-﻿<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

```

Comment:
Update form an updated master and use what master has as the base line I guess sorry!